### DISCUSSION TITLE
APACHE version 2.4.60-240701 causes sites to get downloaded instead of opened.

### DISCUSSION THREAD

**Discussion by MGeurts** [leokhoa/laragon#819] [Q&A]
When clicking the **Web** button or opening sites served by LARAGON, index.php gets downloaded instead of opening the URL.
It doesn't matter what browser I have as default browser.
This also happens with a fresh installed LARAGON.

Any ideas how to solve this problem?

Thanks.

**Comment by MGeurts** [comment]
Found the solution myself. This had nothing to do with LARAGON.
I had to do a **system restore** on my Windows 11 to revert back to a system backup made last week.

Then I remember the problem started after updating Apache from version 2.4.59-240605-win64-VS17 to version **2.4.60-240701-win64-VS17**.
So I updated again and the problem reappeared.

**So do not use Apache version 2.4.60 !!**

**Comment by MGeurts** [comment]
Maybe the developer has to adjust the LARAGON scripts to incorporate this new APACHE version...

**Reply by luisAntonioLAGS** [reply]
I agree with you, I think Laragon is already outdated and we still don't have a new version :c

**Reply by MGeurts** [reply]
I never sead Laragon was outdated. Laragon is very usefull, even today.

**Reply by luisAntonioLAGS** [reply]
> I never sead Laragon was outdated. Laragon is very usefull, even today.

What I'm getting at is that as time passes we cannot update the tools, such as mysql 8.4 and 9, now even apache

**Comment by virusphp** [comment]
same me to with apache newst

**Comment by rpascolo** [comment]
Solved by httpd-2.4.61-240703-win64-VS17

**Reply by jibon2016** [reply]
how to download this version, i can't find this version

**Reply by Pen-y-Fan** [reply]
See [\[Tutorial\] How to add another Apache version](https://github.com/leokhoa/laragon/wiki/%5BTutorial%5D-How-to-add-another-Apache-version)

**Comment by MGeurts** [comment]
I can confirm. The version httpd-2.4.61-240703-win64-VS17 works as expected.

**Comment by EriickDiiaz** [comment]
Confirmed! It works with httpd-2.4.61-240703-win64-VS17

**Comment by jibon2016** [comment]
Not Solve in my PC 

![image](https://github.com/user-attachments/assets/097c2463-32fb-45ae-b979-90a6d6c2330f)

Iam use apache httpd-2.4.61-240703-win64-VS17 version 

![image](https://github.com/user-attachments/assets/9c80bf65-0496-4414-b069-b570d33eaeef)

**How can i solve this problem?**

**Reply by jibon2016** [reply]
@MGeurts Bro please help me..

**Comment by MGeurts** [comment]
Download [httpd-2.4.62-240904-win64-VS17.zip](https://www.apachelounge.com/download/VS17/binaries/httpd-2.4.62-240904-win64-VS17.zip) from [https://www.apachelounge.com/download/](https://www.apachelounge.com/download/)

**Reply by jibon2016** [reply]
I am also try this version but got same

**Reply by jibon2016** [reply]
Use all of these version, i have new window install in my pc but getting the same download file,  also try virtual host "test" to another name but not solved

![image](https://github.com/user-attachments/assets/387926c4-607f-441b-ac94-8b133a9072f4)

**Comment by MGeurts** [comment]
- Do exactly as I wrote. Use the URLs I provided to download.
  Version 2.4.62 should have the date **240904** in the filename.
  
- What version of PHP are you using?
  I use [php-8.4.1-nts-Win32-vs17-x64.zip](https://windows.php.net/downloads/releases/php-8.4.1-nts-Win32-vs17-x64.zip) from [https://windows.php.net/downloads/releases/](https://windows.php.net/downloads/releases/)
   

![LARAGON](https://github.com/user-attachments/assets/c40619ce-38ad-45b6-8fc8-0be8ca65d194)

**Comment by MGeurts** [comment]
Also check if you have the latest RTE installed on your machine.
You need them if I want to use VS17 in LARAGON.

![LARAGON-RTE](https://github.com/user-attachments/assets/0affd6c3-7114-428e-bc41-45868c1e963b)

**Comment by MGeurts** [comment]
Also do not forget to restart after changing assets.
Also do **Tools -> Path -> Remove Laragon from Path** followed by **Tools -> Path -> Add Laragon to Path**.

**Comment by MGeurts** [comment]
**Tip** : Use [https://www.marticliment.com/unigetui/](https://www.marticliment.com/unigetui)/ to keep your Windows system up to date.
